Remove legacy tokens and blueshift adjustment

tf-4 opened the pull request:
## About The Pull Request

Back again. Removes weapon tokens from the armoury, as well as the Armadyne weapons dispensary.

Also removes that one shocked grille in maint that people die to never to be found. It's not fun.

## How This Contributes To The Skyrat Roleplay Experience

Remove obsolete content and noob trap

## Changelog
:cl:
fix: Old sidearms are unobtainable once more, as they should be. Armoury discrepancies have been normalised.
qol: That one shocked grille has been removed from Blueshift. You know the one.
/:cl:

GoldenAlpharex reviewed (approved):
Looks good to me. Usually I'd argue to leave the shocked grille there, but knowing from experience that it's in a part of maintenance people rarely walk through, you're nearly guaranteed to stay dead there basically forever.

tf-4 commented:
> Looks good to me. Usually I'd argue to leave the shocked grille there, but knowing from experience that it's in a part of maintenance people rarely walk through, you're nearly guaranteed to stay dead there basically forever.

pretty much.

Though since your approval I replaced the gun vendor in blueshift as I didn't realise the primary and energy tokens are actually still obtainable, as well as the black-market trader getting a black-market token. Also removed all tokens on Kilo to replace with CUCKS as it got missed in that update.
